libflux/flux-core/src/semantic/vectorize.rs

```diff
@@ -66,6 +66,16 @@ impl Expression {
     fn vectorize(&self, env: &VectorizeEnv<'_>) -> Result<Self> {
         Ok(match self {
             Expression::Identifier(identifier) => {
+                let name = identifier.name.clone();
+                if env.config.features.contains(&Feature::VectorizedConst)
+                    && (name == "true" || name == "false")
```

Comment:
Yeah, this makes sense. I'll want to do the same for `float`. Originally (for float) I tried to check the `file` field on `loc` but it was empty for some cases so I removed the check.

If I understand correctly, for the tests this will mean moving things from `env` to `importer` so that the package provenance is maintained.

Not sure what's needed to copy stuff over to `env` to approximate the prelude. Will investigate today.

Reply:
Had to work backwards from a regular script run, then make the test setup match how the symbols present in real life. Unexpectedly, the booleans are in the `boolean` package! I thought they were listed under universe, but no...